chart/kubeapps/templates/kubeops/rbac.yaml

```diff
@@ -55,8 +55,7 @@ subjects:
 apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
 kind: ClusterRole
 metadata:
-  name: "kubeapps:controller:kubeops-ns-discovery-{{ .Release.Namespace }}"
-  namespace: {{ .Release.Namespace | quote }}
+  name: "kubeapps:controller:kubeops-ns-discovery"
```

Comment:
Again, great to remove the namespace metadata, but the intent of the name is to identify the ClusterRole created by the kubeapps installation in the `.Release.Namespace` namespace.

Reply:
gotcha. will do